canonical-web-and-design/maas-ui
# Delete scripts

## Note
This is dependent on [#183](https://github.com/canonical-web-and-design/maas-ui/pull/183). Leaving this on WIP until 183's merged, to make this easier to review.

## Done
* Implement deleting scripts.

## QA
* Visit testing/commissioning scripts.
* Ensure default scripts (those provided by MAAS) cannot be deleted.
* Upload a testing or commissioning script via old maas, revisit the testing/commissioning page in maas-ui to update the list.
* Delete the script, you should see a confirmation and the script should disappear from the list.
